frappe/desk/search.py

```diff
@@ -267,7 +267,7 @@ def to_string(parts):
 			if len(item) >= 3 and item[2] == label:
 				# remove redundant title ("label") value
 				del item[2]
-			results.append({"value": item[0], "label": label, "description": to_string(item[1:])})
+			results.append({"value": item[0], "label": label or item[0], "description": to_string(item[1:])})
```

Comment:
Looking at the previous line, I wonder if its a better idea to add the "label" key only if label exists